intelmq/bots/BOTS

```diff
@@ -1112,30 +1112,14 @@
             "module": "intelmq.bots.parsers.phishtank.parser",
             "parameters": {}
         },
-        "ShadowServer Chargen": {
-            "description": "ShadowServer Chargen Parser is the bot responsible to parse the report and sanitize the information.",
-            "module": "intelmq.bots.parsers.shadowserver.chargen_parser",
-            "parameters": {}
-        },
-        "ShadowServer Drone": {
-            "description": "ShadowServer Drone Parser is the bot responsible to parse the report and sanitize the information.",
-            "module": "intelmq.bots.parsers.shadowserver.drone_parser",
-            "parameters": {}
-        },
-        "ShadowServer Microsoft Sinkhole": {
-            "description": "ShadowServer Microsoft Sinkhole Parser is the bot responsible to parse the report and sanitize the information.",
-            "module": "intelmq.bots.parsers.shadowserver.microsoft_sinkhole_parser",
-            "parameters": {}
-        },
-        "ShadowServer QOTD": {
-            "description": "ShadowServer QOTD Parser is the bot responsible to parse the report and sanitize the information.",
-            "module": "intelmq.bots.parsers.shadowserver.qotd_parser",
-            "parameters": {}
-        },
-        "ShadowServer SNMP": {
-            "description": "ShadowServer SNMP Parser is the bot responsible to parse the report and sanitize the information.",
-            "module": "intelmq.bots.parsers.shadowserver.snmp_parser",
-            "parameters": {}
+        "ShadowServer": {
+            "description": "ShadowServer Parser is a bot capable of parsing all shadowserver feeds, depending on configuration files. Parameter 'feedname' is used as identifier to chose the correct mapping.",
+            "module": "intelmq.bots.parsers.shadowserver.parser",
+            "parameters": {
+                "feedcode": "",
```

Comment:
---
Mobile

> On 07.07.2016, at 08:44, Dustin Demuth <notifications@github.com> wrote:
> 
> In intelmq/bots/BOTS:
> 
> > -            "parameters": {}
> > -        },
> > -        "ShadowServer QOTD": {
> > -            "description": "ShadowServer QOTD Parser is the bot responsible to parse the report and sanitize the information.",
> > -            "module": "intelmq.bots.parsers.shadowserver.qotd_parser",
> > -            "parameters": {}
> > -        },
> > -        "ShadowServer SNMP": {
> > -            "description": "ShadowServer SNMP Parser is the bot responsible to parse the report and sanitize the information.",
> > -            "module": "intelmq.bots.parsers.shadowserver.snmp_parser",
> > -            "parameters": {}
> > +        "ShadowServer": {
> > +            "description": "ShadowServer Parser is a bot capable of parsing all shadowserver feeds, depending on configuration files. Parameter 'feedname' is used as identifier to chose the correct mapping.",
> > +            "module": "intelmq.bots.parsers.shadowserver.parser",
> > +            "parameters": {
> > +                "feedcode": "",
> IMHO feedcode is used to set an alternative name for a feed, for instance in order to obfuscate the real feed name.
> 
> From my understanding this makes most sense in a Collector, but the shadowserver parse has the ability to override this seeting.
> 
> The parameter is not important for the parser
> 

Agreed. Should we take it out from the parser?


> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>